src/main/java/com/otya/care/user/CareController.java

```diff
@@ -32,13 +32,16 @@ public ResponseEntity<?> createCare(@RequestBody CareForm form) throws CareDupli
         String gender = form.getGender();
         int age = form.getAge();
         String address = form.getAddress();
-        String care_needs = form.getCare_needs();
+        String care_needs = form.getCareNeeds();
```

Comment:
ここも修正しましょう！
```suggestion
        String careNeeds = form.getCareNeeds();
```

あと、まだまだスネークケースになってる変数名と関数名がありますので、探してください！

Reply:
申し訳ありません。
ここの部分は今し方、見つけて修正しました。
他の部分も探してみます。

Reply:
`findByname`もおかしいですね。
`findByName`という形で、単語の頭は大文字にしましょう！

Reply:
修正が一通り終わったら、git commitして、git pushしてくださいね！

Reply:
他クラスのスネークケースを修正しました。
また、findBynameをfindByNameに変更しました。
Mappreのcare_userなどは編集しておりません。
2671533

Reply:
@otyacazyu 
> Mappreのcare_userなどは編集しておりません。

は、もしかして、SQLの`care_users`を指していますか？
それであれば、SQLはJavaとは分けて考えるべきなので、スネークケースのままで大丈夫ですよ！